servers/cromwell/jobs/controllers/jobs_controller.py

```diff
@@ -29,9 +29,13 @@ def abort_job(id):
 
     :rtype: None
     """
+    auth_token = request.headers.get('Authentication')
```

Comment:
This is going to keep the `Bearer ` piece on. Is that intended? i.e. does cromwell expect a bearer auth token?

Reply:
Yes, cromwell-as-a-service expects a bearer auth token